process/05 development/fdd/develop-object-model.md

```diff
@@ -1,7 +1,7 @@
 * 25-word purpose statement
 * Identify the top 10 features
 * or High Value Scenarios (Scenarios for conducting business and determining results)
-* [Four kinds of features](four-kinds-of-features)
+* [Four kinds of features](four-kinds-of-features)  <--- What think link shall point to?
```

Comment:
This page was never migrated from the freelance playbook:

https://github.com/nicholasjhenry/firsthand-playbook/blob/master/Four-kinds-of-features.md

Reply:
What is the best approach here? Migrate the page and link to it or some other way of resolving it is preferable?

Reply:
Just copy the page and link would be fine. Thanks!